README.md

```diff
@@ -91,7 +97,40 @@ reserver --name sample_name1 sample_name2 --token=PYPI_TOKEN --test
 ```console
 reserver --name sample_name1 sample_name2 --token=PYPI_TOKEN
 ```
-
+You can customize package parameters (listed below) by passing considered params as JSON file(s).
+
+Customizable package parameters are: (default value is written in front of them)
```

Comment:
I suggest showing parameters in a table instead of this style.
For example something like:

<table>
<tr>
<th>Parameter</th>
<th>Type</th>
<th>Default</th>
</tr>
<td><code>description</code></td>
<td>string</td>
<td><code>This name has been reserved using Reserver</code></td>
</tr>
</table>